### PR TITLE
Refactor helpers to clear deprecation warnings for Ember 2.0

### PR DESCRIPTION
**Pull request by MichaelVdheeren**

### CODE REVIEW

File: tests/unit/helpers/format-number-test.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ test("formats the with the default configuration", function(assert) {
   assert.equal(callHelper(formatNumber, 123.4567, {}), "123");
 });
 
-test("formats the with options passed to the handlebars templace", function(assert) {
+test("formats the with options passed to the handlebars templact", function(assert) {
```
Comment (josemarluedke) [inline tests/unit/helpers/format-number-test.js]: Typo. :D

**Comment by josemarluedke** [comment]
Please :shipit:.

**Comment by cibernox** [comment]
I'll merge now and make some adjustements, because I plan to bump a mayor version number an make it only compatible with ember 1.13 and above. 

Hold on a couple hours.

**Comment by josemarluedke** [comment]
Thank you @cibernox.

**Pull request merged**